AhjoApiService/AhjoToStorageMapper.cs

```diff
@@ -35,5 +36,17 @@ public static List<StorageMeetingDTO> CreateStorageMeetingDTOs(List<AhjoMeetingD
             return result;
         }
 
+        private static string GetLanguageFromHtml(string html)
+        {
+            if(html.Length > 19)
```

Comment:
magic number, `const string LangInfoStringLength = 20` or something? and `if (html.length >= LangInfoStringLength)` space between `if` and `(`

Reply:
fixed